src/components/RedoIcon.tsx

```diff
@@ -4,19 +4,43 @@ import { token } from '../../styled-system/tokens'
 import Icon from '../@types/Icon'
 
 /** A redo icon. */
-const RedoIcon = ({ fill, size = 18, style, className }: Icon) => {
+const RedoIcon = ({ fill, size = 14, style = {}, className }: Icon) => {
+  const scalingFactor = 1.37
+  const newSize = size * scalingFactor
+  const strokeColor = style.fill || fill || token('colors.fg')
+
   return (
     <svg
-      version='1.1'
       className={cx(icon(), className)}
       xmlns='http://www.w3.org/2000/svg'
-      width={size}
-      height={size}
+      width={newSize}
+      height={newSize}
+      viewBox='0 0 24 24'
       fill={fill || token('colors.fg')}
-      style={style}
-      viewBox='0 -50 600 600'
+      style={{ ...style, width: `${newSize}px`, height: `${newSize}px`, marginTop: '-1px' }}
```

Comment:
This -1px marginTop might be affecting the positioning across font sizes since it does not scale. You can adjust the font size using the links in the footer:
<img width="221" alt="image" src="https://github.com/user-attachments/assets/cdbf9240-ee4e-42d7-9cc7-8dc55e6294ca">

I tested the vertical centering by scrolling the toolbar a bit to the right (and I temporarily deleted the padding on the scroll arrow for closer examination).

## This PR

Font Size 18 (default):
<img width="105" alt="Brave Browser 2024-09-28 17 56 02" src="https://github.com/user-attachments/assets/08890585-6b36-42ae-99d3-8c90b22ec189">

Font Size 25:
<img width="124" alt="Brave Browser 2024-09-28 17 55 18" src="https://github.com/user-attachments/assets/c92e7c41-b74b-47bb-96a8-cb089e96f91a">

# `main`

While the icons are admittedly slightly lower than center, the difference is not amplified when the font size increases.

Font Size 18 (default):
<img width="91" alt="Brave Browser 2024-09-28 17 59 34" src="https://github.com/user-attachments/assets/3704f4ed-8602-4298-b766-e4fe15197dc9">

Font Size 25:
<img width="114" alt="Brave Browser 2024-09-28 17 59 48" src="https://github.com/user-attachments/assets/68b9d25a-f35a-43fb-b602-2ac264e04336">

Note that I'm not sure if it is the toolbar button icons or the arrow icon is wrong, but either way they need to be aligned vertically along their centers regardless of font size.

Let me know if that makes sense!

Reply:
The new icons are centered within the SVG and include some padding, while the previous icons were aligned to the top-left corner. I aimed to position the new icons at the same level as the previous ones for consistency. Scaling with state.fontSize is part of Milestone 3, so for now, we're focused on achieving perfect positioning without scaling. A good option for now would be to remove the margin-top: -1px entirely, as all the icons will be aligned on the same level after replacement.

Reply:
Great! We'll revisit in Milestone 3.